src/main/java/dev/jbang/dependencies/DependencyResolver.java

```diff
@@ -65,7 +65,7 @@ public DependencyResolver addClassPaths(String classPaths) {
 
 	public ModularClassPath resolve() {
 		ModularClassPath mcp = DependencyUtil.resolveDependencies(
-				new ArrayList<>(dependencies), new ArrayList<>(repositories),
```

Comment:
this can't be right  - the sequence of maven repositoreis matters.

Reply:
Sure it's right :-)

They're LinkedHashSets, they retain their ordering: https://github.com/jbangdev/jbang/blob/1cf630cd600960f771aaf6e7b9bb14f0283341e8/src/main/java/dev/jbang/dependencies/DependencyResolver.java#L16-L18